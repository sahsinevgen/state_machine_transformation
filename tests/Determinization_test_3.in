5 
1
2
4 5

8
1 1 a
1 2 a
2 2 b
2 3 b
3 4 #
3 5 #
4 4 c
5 5 a