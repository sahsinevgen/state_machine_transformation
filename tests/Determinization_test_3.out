6
1
3
4 5 6 
18
1 2 a
1 3 b
1 3 c
2 2 a
2 4 b
2 3 c
3 3 a
3 3 b
3 3 c
4 5 a
4 4 b
4 6 c
5 5 a
5 3 b
5 3 c
6 3 a
6 3 b
6 6 c